src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,8 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
```

Comment:
no need to create `Lottery` instance each time when you call this method, consider extracting it to the `private static final ` class-level field